semantic_conventions/resource/faas.yaml

```
@@ -16,30 +16,38 @@ groups:
+            followed by a forward slash followed by the function name (this form
+            can also be seen in the resource JSON for the function).
+            This means that a span attribute MUST be used, as an Azure function
+            app can host multiple functions that would usually share
+            a TracerProvider (see also the `faas.id` attribute).
+        examples: ['my-function', 'myazurefunctionapp/some-function-name']
       - id: id
         type: string
         brief: >
           The unique ID of the single function that this runtime instance executes.
         note: |
-          Depending on the cloud provider, use:
+          On some cloud providers, it may not be possible to determine the full ID at startup,
+          so consider setting `faas.id` as a span attribute instead.
+
+          The exact value to use for `faas.id` depends on the cloud provider:
 
           * **AWS Lambda:** The function [ARN](https://docs.aws.amazon.com/general/latest/gr/aws-arns-and-namespaces.html).
-          Take care not to use the "invoked ARN" directly but replace any
-          [alias suffix](https://docs.aws.amazon.com/lambda/latest/dg/configuration-aliases.html) with the resolved function version, as the same runtime instance may be invokable with multiple
-          different aliases.
+            Take care not to use the "invoked ARN" directly but replace any
+            [alias suffix](https://docs.aws.amazon.com/lambda/latest/dg/configuration-aliases.html)
+            with the resolved function version, as the same runtime instance may be invokable with
+            multiple different aliases.
           * **GCP:** The [URI of the resource](https://cloud.google.com/iam/docs/full-resource-names)
-          * **Azure:** The [Fully Qualified Resource ID](https://docs.microsoft.com/en-us/rest/api/resources/resources/get-by-id) of the invoked function, *not* the function app,
-          having the form `/subscriptions/<SUBSCIPTION_GUID>/resourceGroups/<RG>/providers/Microsoft.Web/sites/<FUNCAPP>/functions/<FUNC>`. This means that a span attribute MUST
-          be used, as an Azure function app can host multiple functions that would usually share
-          a TracerProvider (see also the paragraph below).
-
-          On some cloud providers, it may not be possible to determine the full ID at startup,
-          so consider setting `faas.id` as a span attribute instead.
+          * **Azure:** The [Fully Qualified Resource ID](https://docs.microsoft.com/en-us/rest/api/resources/resources/get-by-id) of the invoked function,
+            *not* the function app, having the form
+            `/subscriptions/<SUBSCIPTION_GUID>/resourceGroups/<RG>/providers/Microsoft.Web/sites/<FUNCAPP>/functions/<FUNC>`.
+            This means that a span attribute MUST be used, as an Azure function app can host multiple functions that would usually share
+            a TracerProvider (see also the paragraph below).
```

```suggestion
            a TracerProvider.
```

I guess we don't need this anymore right? After the re-shuffling of paragraphs?